### DISCUSSION TITLE
Bullets and numbers not rendering

### DISCUSSION THREAD

**Discussion by addorai** [outline/rich-markdown-editor#372] [Q&A] [answered]
The symbols for bullets and numbers aren't showing up for some reason (check out the screenshot). The first list is bullets and the second is numbers. It works for checklists. 

It's still saving correctly (and also works on copy + paste), just not rendering the bullet symbol and numbers within the editor. 

Any ideas? Thanks! 

![image](https://user-images.githubusercontent.com/58643152/106216192-9bb62800-61a0-11eb-9d62-871c5d565f93.png)

**Comment by tommoor** [comment, marked as answer]
You've probably overridden `list-style` css somewhere, check the style tab in devtools to see how.

**Reply by addorai** [reply]
Ah, you're right. Thanks!
